lambdas/mail_send.js

```diff
@@ -4,17 +4,47 @@ if (process.env.NODE_ENV === 'development') {
 
 const domain = process.env.MAILGUN_DOMAIN;
 const apiKey = process.env.MAILGUN_API_KEY;
+const secret_key = process.env.SECRET_SEND_KEY;
 
+const { send } = require('micro');
+const statuses = require('statuses');
+const { upload } = require('micro-upload')
 const mailgun = require('mailgun-js');
 
-const mg = mailgun({ apiKey, domain });
-const data = {
-  from: 'WebConf <no-reply@webconf.tech>',
-  to: 'gahs94@gmail.com',
-  subject: 'WebConf • Nuestro newsletter 🙈',
-  text: 'Hola perrix! :D'
-};
+module.exports = upload(async (req, res) => {
```

Comment:
Since you're wrapping everything in an async function, there should be no callback or promises anywhere here. To keep consistency, let's use async/await where possible.